### PR TITLE
refactor(ticket opt-out): Centralize the messaging for the opt out ch…

### PR DESCRIPTION
**Pull request by Camwyn**
…eckboxes. Change text and defau

In a step toward better GDPR compliance, set the opt-out checkboxes to default to checked.
Change the text to be a bit more inclusive(since the setting affects API data as well).
Move the text to `Tribe__Tickets__Privacy` for now to centralize where we set and handle these.
Set up some functionality for future implementation as we shift toward opt-in.
Adds a note to the classic ticket metabox settings panel.
Also hacks in a note about the change into the ticket block settings dashboard.

:camera: http://p.tri.be/gAp0na
:camera: http://p.tri.be/C6iNVN
:camera: http://p.tri.be/58CvLD

:ticket: [ET-615]

Draft awaiting text change verifications

[ET-615]: https://moderntribe.atlassian.net/browse/ET-615

### CODE REVIEW

File: src/views/blocks/tickets/extra-available.php [hunk 1]
```diff
@@ -60,12 +60,12 @@
...
  * @since 4.11.1
  */
-$show_unlimited = apply_filters( 'tribe_tickets_block_show_unlimited_availability', false, $available );
+$show_unlimited = apply_filters( 'tribe_tickets_block_show_unlimited_availability', true, $available );
 ?>
 <div
 	class="tribe-common-b3 tribe-tickets__item__extra__available"
 >
-	<?php if ( $show_unlimited ) : ?>
+	<?php if ( $show_unlimited -1 && $available ) : ?>
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets/extra-available.php]: This doesn't look right

File: src/views/blocks/rsvp/form/opt-out.php [hunk 2]
```diff
@@ -25,12 +25,17 @@
  * @param bool
  */
 $hide_attendee_list_optout = apply_filters( 'tribe_tickets_plus_hide_attendees_list_optout', $modal );
-if ( $hide_attendee_list_optout
-	 && ! class_exists( 'Tribe__Tickets_Plus__Attendees_List' )
-	 && Tribe__Tickets_Plus__Attendees_List::is_hidden_on( $this->get( 'post_id' ) )
+if (
```
Comment (sc0ttkclark) [inline src/views/blocks/rsvp/form/opt-out.php]:
```suggestion

if (
```

File: src/Tribe/Privacy.php [hunk 3]
```diff
@@ -4,17 +4,38 @@
...
+		add_action( 'admin_init', [ $this, 'privacy_policy_content' ], 20 );
+		add_action( 'init', [ $this, 'init_text' ] );
 
-		add_filter( 'wp_privacy_personal_data_exporters', array( $this, 'register_exporters' ), 10 );
-		add_filter( 'wp_privacy_personal_data_erasers', array( $this, 'register_erasers' ), 10 );
+		add_filter( 'wp_privacy_personal_data_exporters', [ $this, 'register_exporters' ], 10 );
+		add_filter( 'wp_privacy_personal_data_erasers', [ $this, 'register_erasers' ], 10 );
+	}
+
+	public function init_text() {
```
Comment (sc0ttkclark) [inline src/Tribe/Privacy.php]: Needs phpdoc

File: src/Tribe/Privacy.php [hunk 4]
```diff
@@ -4,17 +4,38 @@
...
+		$this->opt_out_text = __( "Hide my attendee data from public view.", 'event-tickets' );
```
Comment (sc0ttkclark) [inline src/Tribe/Privacy.php]:
```suggestion
		$this->opt_out_text = __( 'Hide my attendee data from public view.', 'event-tickets' );
```

File: src/Tribe/Privacy.php [hunk 5]
```diff
@@ -4,17 +4,38 @@
...
+		$this->opt_in_text = __( "Allow use of my attendee data publicly on this site.", 'event-tickets' );
```
Comment (sc0ttkclark) [inline src/Tribe/Privacy.php]:
```suggestion
		$this->opt_in_text = __( 'Allow use of my attendee data publicly on this site.', 'event-tickets' );
```

File: readme.txt [hunk 6]
```diff
@@ -135,6 +135,9 @@ Currently, the following add-ons are available for Event Tickets:
 * Fix - Correct broken javascript for themes that change the base post classes. [ET-614]
 * Tweak - Refine logic for the no results notice on the "My Tickets" page. [ETP-151]
 * Tweak - Remove duplicate ticket script to prevent conflicts. [ET-596]
+* Tweak - Change the opt-out checkbox to be checked by default. [ET-615]
```
Comment (sc0ttkclark) [inline readme.txt]: These changelog items need to explain which opt-out checkbox (like public Attendee List opt-out)

Comment (Camwyn) [reply readme.txt]: OK, but it is now "one opt-out to rule them all" 💍

**Comment by Camwyn** [comment]
@sc0ttkclark changes made, will need re-review once tests pass.

**Pull request merged**